ch05/ex5_23.cpp

```diff
@@ -1,11 +1,29 @@
 #include <iostream>
-using namespace std;
+#include <string>
+using std::string;
+using std::cin;
+using std::cout;
+using std::endl;
 
-int main(void)
-{
-	int a, b;
-	cin >> a >> b;
-	cout << static_cast<double>(a) / b << endl;
-	
-	return 0;
-}
\ No newline at end of file
+
```

Comment:
IMHO, It is a little incomprehensible for this question... (Even though the answer is not my contribution...)